### ISSUE TITLE
[high] JAWS+Chrome: Many Pages: The social media links are announced without labels

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5499]
## Action Performed:
1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Tab to focus the "Terms of service" link and press Enter key.
3. Tab to navigate through the social media links at the bottom of the page and listen to the announcements.

## Expected Result:
Each social media link should be announced along with its label, e.g. "Expensify Podcast", "Expensify Twitter", "Expensify Instagram", "Expensify Facebook" and "Expensify Linkedin".

## Actual Result:
Only the link role is announced for the social media links.

HTML:
`<a href="https://we.are.expensify.com/podcast" class="social-media-link expensicons expensicons-podcast" target="_blank"></a><a href="https://www.twitter.com/expensify" class="social-media-link expensicons expensicons-twitter" target="_blank"></a><a href="https://www.instagram.com/expensify" class="social-media-link expensicons expensicons-instagram" target="_blank"></a><a href="https://www.facebook.com/expensify" class="social-media-link expensicons expensicons-facebook" target="_blank"></a><a href="https://www.linkedin.com/company/expensify" class="social-media-link expensicons expensicons-linkedin" target="_blank"></a>`

Other occurrences:
1. The same issue is also seen on Privacy Policy and Licenses pages.

## Workaround:
No workaround

## Area issue was found in:
Terms of service page, Licenses page, Privacy policy page

## Failed WCAG checkpoints
4.1.2

## User impact:
The screen reader users are not informed about the purpose of the social media links.

## Suggested resolution:
Use aria-label attribute to provide a descriptive label to each social media link (e.g. "Expensify Facebook"). See: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA14

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134719095-43bb1925-225c-422a-ab50-122b4bea39a0.mp4

https://user-images.githubusercontent.com/88733897/134719192-7bf14288-534e-4c95-982d-50df5b5976fb.mp4

![Bug5181130_name__role__value_-_social_media_links_-_terms_of_services](https://user-images.githubusercontent.com/88733897/134719210-2d4154df-fcb4-4589-b5c6-d44af6d8964b.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by shawnborton** [comment]
I think this one should be closed as it's not part of the actual NewDot app. Basically the same comment I've had elsewhere. cc @kadiealexander @stitesExpensify
